corehq/apps/app_manager/models.py

```diff
@@ -5154,7 +5152,7 @@ def exclude_form(form):
                 filename = prefix + self.get_form_filename(**form_stuff)
                 form = form_stuff['form']
                 try:
-                    files[filename] = self.fetch_xform(form, build_profile_id=build_profile_id)
+                    files[filename] = form.render_xform(build_profile_id=build_profile_id)
                 except XFormValidationFailed:
```

Comment:
can a validation error still be thrown from `render_xform`?

Reply:
No the thinking here was that `_get_form_files` is only used in a context where the form has already been validated (see `create_all_files`), so removing the additional validation seemed safe and an easy way to _slightly_ speed this up.

Reply:
Then is this try block still necessary?

Reply:
Ah I see. The try block is still necessary because `render_xform`, specifically `exclude_languages` in `add_stuff_to_xform` raises a XFormException, but XFormValidationFailed is only raised in `validate_xform`. See b1fb93369163f150a5bbb8f1c51f9c079de9da57